libcxx/test/std/localization/locale.categories/category.numeric/locale.num.get/facet.num.get.members/get_long_double.pass.cpp

```diff
@@ -193,6 +319,7 @@ int main(int, char**)
         assert(err == ios.failbit);
         assert(v == INFINITY);
     }
+#if LDBL_MANT_DIG >= 64
```

Comment:
Mainly curious why is this change needed after this patch?

Reply:
This is a drive-by change that would make MSVC, whose `long double` is of binary64 format, pass the test.

Reply:
I initially expected that, but these tests are running with clang-cl (and mingw). I expect at least clang-cl uses LDBL_MANT_DIG == 53 and thus give the same results as MSVC STL.

I'm especially puzzled why these infinity tests fail while the infinity test above passes.
The `304888344611713860501504000000` test seems more likely to fail due to a lack of precision.
(I see this test is listed in https://github.com/microsoft/STL/blob/main/tests/libcxx/expected_results.txt)

Did you run these tests with MSVC STL and checked why they failed?

Reply:
Thanks for noticing this! I've confirmed that the conditional compilation has become unnecessary now.